lib/datasets/communities.rb

```diff
@@ -0,0 +1,184 @@
+require "csv"
+
+require_relative "dataset"
+module Datasets
+  class Communities < Dataset
+    Record = Struct.new(
+      :state,
+      :county,
+      :community,
+      :communityname,
+      :fold,
+      :population,
+      :householdsize,
+      :racepctblack,
```

Comment:
Could you use `:race_pct_black`?
BTW, what is `pct`...?

> -- racepctblack: percentage of population that is african american (numeric - decimal)

Percent!!! Could you use `race_percent_black`?